package.json

```diff
@@ -5,11 +5,11 @@
   "author": "Blockly Team",
   "license": "Apache-2.0",
   "scripts": {
-    "audit:fix": "lerna run audit:fix && lerna bootstrap",
-    "boot": "lerna bootstrap",
-    "build": "lerna run build",
-    "clean": "lerna run clean",
-    "clean:node": "lerna clean --yes",
+    "audit:fix": "cd plugins && lerna run audit:fix && lerna bootstrap",
+    "boot": "cd plugins && npm ci",
+    "build": "cd plugins && lerna run build",
+    "clean": "cd plugins && lerna run clean",
+    "clean:node": "cd plugins && lerna clean --yes",
     "deploy:prepare": "npm run deploy:prepare:plugins && npm run deploy:prepare:examples && gulp predeploy",
     "deploy:prepare:examples": "cd examples && npm run predeploy",
     "deploy:prepare:plugins": "npm run clean && lerna run build && lerna run predeploy",
```

Comment:
Looks like this should have a `cd plugins &&` spliced in before the `lerna` invocations, unless I've misunderstood something.

Reply:
Thanks for the catch!